reana-ui/src/components/WorkflowProgressCircleBar.module.scss

```diff
@@ -0,0 +1,41 @@
+@import "@palette";
+
+.progress-bar-container {
+  svg {
+    width: 100%;
+    height: 100%;
+    transform: rotate(-90deg);
+  }
+
+  circle {
+    fill: none;
+    transition: stroke-dasharray 0.35s ease;
+  }
+
+  .progress-bar-running {
+    stroke: $sui-blue;
+  }
+
+  .progress-bar-finished {
+    stroke: $sui-green;
+  }
+
+  .progress-bar-failed {
+    stroke: $sui-red;
+  }
+
+  .progress-bar-background,
+  .progress-bar-workflow-status-queued,
+  .progress-bar-workflow-status-pending,
+  .progress-bar-workflow-status-created {
```

Comment:
I'm not exactly sure what is the purpose of the `.progress-bar-workflow-status-...` classes?

Reply:
Makes sense, thanks!